sdk/webpubsub/Azure.ResourceManager.WebPubSub/api/Azure.ResourceManager.WebPubSub.net8.0.cs

```diff
@@ -385,6 +676,18 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.WebPubSub.Models.PublicNetworkAcls>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.WebPubSub.Models.PublicNetworkAcls>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class PublicTrafficIPRule : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.WebPubSub.Models.PublicTrafficIPRule>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.WebPubSub.Models.PublicTrafficIPRule>
```

Comment:
the base type's name is fine, but this one we might need to prepend the rp name.